source/partials/_header.haml

```diff
@@ -7,4 +7,4 @@
     %nav#js-main-nav.page-head__nav
       %ul
         %li
-          = link_to("Submit a Talk", "/submit")
+          = link_to("Submit a Talk", "/submit.html")
```

Comment:
I believe this is going to break in production. We can fix this locally by using Middleman’s “pretty directory” option.